include/cantera/kinetics/Kinetics.h

```diff
@@ -581,7 +581,7 @@ class Kinetics
     //!  - `_ddX`: derivative with respect to species mole fractions (a matrix)
     //!
     //! Settings for derivative evaluation are set by keyword/value pairs using
-    //! the methods @see getDerivativeSettings and @see setDerivativeSettings.
+    //! the methods @see getDerivativeSettings and setDerivativeSettings.
```

Comment:
I think all you need here is 
```suggestion
    //! the methods getDerivativeSettings() and setDerivativeSettings().
```

Reply:
Done. I figure that the parentheses are required to detect methods in all cases? It may make sense to go through the documentation in a separate PR as they are often left off when used in combination with `@see`? (although doxygen may be smart enough to infer things in that case)

Reply:
Yeah, that's correct. The `@see` notation is just for creating a `see also` block within the documentation.

I think there are probably a lot of cases where the parens are missing when referencing functions, both with and without `@see`.